lib/src/widgets/image_message_view.dart

```diff
@@ -139,6 +148,17 @@ class ImageMessageView extends StatelessWidget {
                 reaction: message.reaction,
                 messageReactionConfig: messageReactionConfig,
               ),
+            if (!showMessageTimeIn.isOnRightSwipe &&
+                !showMessageTimeIn.isDisable)
+              Positioned(
+                right: message.reaction.reactions.isNotEmpty ? 16 : 18,
```

Comment:
this static value might break something, please assure all the possible scenarios.